ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/JumpPass.scala

```diff
@@ -0,0 +1,38 @@
+package io.joern.ghidra2cpg.passes
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.EdgeTypes
+import io.shiftleft.codepropertygraph.generated.nodes.Method
+import io.shiftleft.passes.{DiffGraph, IntervalKeyPool, ParallelCpgPass}
+import io.shiftleft.semanticcpg.language._
+
+class JumpPass(cpg: Cpg, keyPool: IntervalKeyPool)
+  extends ParallelCpgPass[Method](
+    cpg,
+    keyPools = Some(keyPool.split(1))
+  ) {
+
+  override def partIterator: Iterator[Method] = cpg.method.l.iterator
+
+  override def runOnPart(method: Method): Iterator[DiffGraph] = {
+  implicit val diffGraph: DiffGraph.Builder = DiffGraph.newBuilder
+    method.call
+      .nameExact("<operator>.goto")
+      .where(_.argument.order(1).isLiteral)
+      .foreach { sourceCall =>
+        sourceCall.argument.order(1).code.l.headOption match {
+          case Some(destinationAddress) =>
+            method.call.lineNumber(Integer.parseInt(destinationAddress, 16)).foreach {
```

Comment:
Somewhat nitpicky comment left over from previous CR. Parsing the address can lead to the program crashing, so a nice way to avoid it is by doing something like
```
def parseAddress(address: String): Option[Int] {
  Try(Integer.parseInt(address)).toOption
}

// and then
sourceCall.argument.order(1).code.l.headOption.flatMap(parseAddress) match {
  ...
```